apps/api/src/server/applications/application/project-updates/__tests__/project-updates.test.js

```diff
@@ -12,6 +12,7 @@ import { EventType } from '@pins/event-client';
 import { eventClient } from '../../../../infrastructure/event-client.js';
 import { htmlContentError, statusError } from '../project-updates.validators.js';
 import { NotFound } from '#utils/api-errors.js';
+import { jest } from '@jest/globals';
```

Comment:
line 1, the import was red-lining for me, poss it has changed location? I needed to change to
import { EventType } from '@pins/event-client/src/event-type';

Reply:
Good spot. The import is correct, unfortunately the types for that module aren't quite right, so vscode can't pick it up. I'll try and sort that separately.